roles/installer/defaults/main.yml

```diff
@@ -278,10 +278,10 @@ init_container_extra_commands: ''
 init_container_extra_volume_mounts: ''
 
 ee_images:
-  - name: AWX EE (latest)
-    image: quay.io/ansible/awx-ee:latest
+  - name: "AWX EE ({{ _image_version }})"
+    image: "quay.io/ansible/awx-ee:{{ _image_version }}"
```

Comment:
or
```suggestion
  - name: "AWX EE (latest)"
    image: "quay.io/ansible/awx-ee:{{ _image_version }}"
  - name: "AWX EE ({{ _image_version }})"
    image: "quay.io/ansible/awx-ee:{{ _image_version }}"
```

Reply:
It's a tough decision.  I generally like the PR.

On one hand, users who understand the implications and how they should be used would like both EE's to be  added (versioned and latest).  

However, sometimes it is better to be opinionated, in which case, we'd want to go with the versioned EE because that is what was tested with that version of AWX...
I lean towards loading both EE's. But I like the idea of polling the community on this PR and in Matrix to get feedback on which is desired.

Reply:
The first item on this list will be adopted as the global default EE for the job templates that has no default EE. Upgrading will not remove existing old EEs, but the global default EE should always be replaced by the first one on this list upon upgrade.
Therefore, if the top of this list is the versioned image, existing jobs may start running on the new EE image after the upgrade.

In the future, I would like to remove the `latest` tag from this list.
The outdated `latest` images or the difference between the `latest` images cached on each nodes in the cluster is asking for trouble. If a job starts using a new EE and starts to fail, it is easier to troubleshoot and do a workaround if the versioned image is used rather than `latest`. I want to remove the question such as "How to renew `latest` iamge?" from the FAQ on the community.

However, since this is the first release where the EE image version is bound, at this stage, to reduce users confusion, I vote for keeping both `latest` and versioned.
 Also I would like to remove `latest` in 24.x or 25.x.

Reply:
If, then this seems better that previous suggestion because it leaves the auto-upgradeable "latest" ee yet still creating new ones with frozen version tags. When set on organisation or project level this would achieve my goal by preserving EE version through upgrade and release of new version. If someone wants to be on cutting edge, can use "latest" tag, otherwise keep to frozen tags. This would also create new EE entry with each new AWX version. I would choose that over the previous suggestion.

Please also keep in mind, that all previous versions already out there (before this PR eventually gets merged) will still have "latest" tag EE version that would not be removed nor changed. Therefore this solution even better fits the case